### ISSUE TITLE
Filter-bar (Combo-Box) :  Screen reader should convey ‘Checked’ information for the checked checkbox and ‘Unchecked’ information for the unchecked checkbox both the information should be avoided.

### ISSUE THREAD

**Issue by ashish-C5339954** [SAP/ui5-webcomponents-react#5474]
### Describe the bug

Checked and Unchecked info is not read by JAWS ( screeen reader) while enabling or disabling some value in the combo-box dropdown .
bug-issue link -> https://jira.eurekacloud.io/browse/CRP-91673

### Reproduction steps

Issue Type: Manual

Observed Behavior: 
In entire application while navigating around the checkboxes within ‘Filters’ dialog, both ‘Checked and unchecked’ information is being read.
deleted

### Expected Behaviour

Screen reader should convey ‘Checked’ information for the checked checkbox and ‘Unchecked’ information for the unchecked checkbox both the information should be avoided.

### Screenshots or Videos
deleted

### UI5 Web Components for React Version

0.27.5

### UI5 Web Components Version

1.6.0

### Browser

Chrome

### Operating System

Windows 11, version 23H2

### Declaration

- [X] I’m not disclosing any internal or sensitive information.

**Comment by Lukas742** [comment]
Hi @ashish-C5339954 

this is a public repository. Please don't link to internal systems or share internal documents. I removed both document links even though I think there weren't any compromising information shared. Better safe than sorry. If you need to share information with us, that can't made be public, then contact us internally.

___

Your using a version that is pretty old and still was in beta status, since then the FilterBar received major refactorings, so unfortunately we can't offer support for an outdated version. Please update your dependencies and then check again.
